### DISCUSSION TITLE
Setting Up Dev Env, Connect thrift to thirdparty/installed/bin

### DISCUSSION THREAD

**Discussion by thaotran27** [apache/doris#35611] [A - General / Q&A]
Regarding this from the Setting up Dev Env for Intellij Guide: "If you only develop fe without compiling thirdparty, you need to install thrift, and copy or connect thrift to the thirdparty/installed/bin directory", I am not sure how to do this step.

Details about what I have done and my system:

- OS: Windows, currently using Docker to have Ubuntu
- What I have done: follow along all the steps to install thrift on Ubuntu, resulting in a folder "thrift-0.16.0". I have checked the version of thrift and it is correctly installed.
- How do I copy or connect thrift to the thirdparty/installed/bin folder?

Thank you so much for taking the time to help me!

**Comment by ixzc** [comment]
you can use binary package
https://doris.apache.org/zh-CN/download
